### PR TITLE
Internal: Add Troubleshooting section to Readme

### PR DESCRIPTION
**Pull request by rlingineni**
Adding a troubleshooting section here that links to our development FAQ

### CODE REVIEW

**Review by AlbertCarreras — APPROVED** [review]

**Comment by AlbertCarreras** [comment]
@rlingineni Run prettier.

**Comment by rlingineni** [comment]
> @rlingineni Run prettier.

Darn, I made this change from the browser. I'll fix as soon I as get a chance.

You know, for these autofixable things, I'm wondering if I can add a Github Action hook that runs prettier and re-commits.

Let me look into it

File: README.md [hunk 1]
```diff
@@ -91,3 +91,8 @@ Or via yarn:
 ```bash
 yarn add @types/gestalt
 ```
+
+### Troubleshooting
+
+Take a look at our [FAQ](https://gestalt.pinterest.systems/get_started/faq) section if you run into any development problems
```
Comment (dangerismycat) [inline README.md:97]: super nit: this should end with a period?

**Review by dangerismycat — APPROVED** [review]

**Comment by dangerismycat** [comment]
@rlingineni Do you plan to fix/merge this PR? Otherwise, do you mind closing it? Thank you! 🙂

**Pull request merged**